### PR TITLE
feat- sign-up page added

### PR DESCRIPTION
**Pull request by ImrozNezami**
Sign-up page
Screenshot (239)

### CODE REVIEW

**Review by Priyanshi-Tiwari — APPROVED** [review]
The code looks good and clean to me.

File: stylesheet/sign-up.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+}
+.nav-bar {
+  margin-bottom: 5rem;
+  padding: 0px;
+  margin-top: 0px;
+}
+.signup-container {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
```
Comment (nakulsharma15) [inline stylesheet/sign-up.css:14]: Just a suggestion, You can use a utility class for it

**Review by nakulsharma15 — APPROVED** [review]
Gave a small suggestion. Everything else looks good to me

File: stylesheet/sign-up.css (hunk 1 above)
Comment (ImrozNezami) [reply stylesheet/sign-up.css:14]: Okay

**Pull request merged**